README.md

```diff
@@ -181,7 +181,7 @@ BENCHMARK_TEMPLATE(BM_Sequential, WaitQueue<int>)->Range(1<<0, 1<<10);
 Three macros are provided for adding benchmark templates.
 
 ```c++
-#if __cplusplus >= 201103L // C++11 and greater.
+#ifdef BENCHMARK_HAS_CXX11 // C++11 and greater.
```

Comment:
you probably don't need the comment now :)

Reply:
Indeed.